New package: PRAS v0.7.0

JuliaRegistrator opened the pull request:
- Registering package: PRAS
- Repository: https://github.com/NREL/PRAS
- Created by: @GordStephen
- Version: v0.7.0
- Commit: ffb58a54145d3f40b385817d737d691bbbbcec4a
- Reviewed by: @GordStephen
- Reference: https://github.com/NREL/PRAS/commit/ffb58a54145d3f40b385817d737d691bbbbcec4a#commitcomment-150179467
- Description: NREL's Probabilistic Resource Adequacy Suite (PRAS)
<!-- 723ffce5a574ab95236c1e8ac347ee92f3733613d647d8d2c2809b18c54e6600dca4d707397128835dfe95999b49634af06ac2a0f9c60efbc4ee45776fe5bece7a9fe865189436c3e521a23ad4d8b0abead532491f143c0c0a4968d144f8668e30dc9bf4bedf70c6af554253d380b765cccfd4ae141140fbbe7152681f74330e1df20ba5af64a24614eb447f7144a359cf16b3023ed0ad94f5e300b95d1c41e87bc216aa544776581487067f518ee6c8cf83a4244c3aa0605daefa8dd20addcc5d0b119578ce3b15431a2f44c0aa5ed0716aa8e042a0096dcff88dc52461a979 -->

GordStephen commented:
Note that the automerge checks won't pass because of the package name, but consensus in https://github.com/JuliaRegistries/General/pull/120172 and https://github.com/JuliaRegistries/General/pull/120786 seems to be that we can get an exception for this?

GordStephen commented:
Whoops, looks like we're missing compat bounds for Reexport, let me fix that.

JuliaRegistrator committed:
New package: PRAS v0.7.0

UUID: 05348d26-1c52-11e9-35e3-9d51842d34b9
Repo: https://github.com/NREL/PRAS.git
Tree: 31fed7232c439c8def5897a4bcb788e72c3f3ec2

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
